site/content/installation.md

```diff
@@ -1,38 +1,40 @@
 
-* [Jupyter Notebook](https://jupyter.org/) comes installed with Anaconda, and is similar to RMarkdown. Jupyter notebooks are web-based applications that let you edit and run blocks of code along with markdown text, equations, and visualizations in a single notebook file. 
-* [Spyder](https://www.spyder-ide.org/) also comes installed with Anaconda and is most similar to RStudio for R. It offers editing, analysis, and debugging support for Python. 
-* [Visual Studio Code](https://code.visualstudio.com/) (or VS Code) can be installed through Anaconda, and includes support for debugging, syntax highlighting, code completion, and more. It also offers thousands of extensions (including [Python-specific extensions](https://code.visualstudio.com/docs/languages/python), a [Quarto extension](https://quarto.org/docs/tools/vscode.html), and an [AWS extension](https://aws.amazon.com/visualstudiocode/)).  
+VSCode includes support for debugging, syntax highlighting, code completion, and more. It also offers thousands of extensions (including [Python-specific extensions](https://code.visualstudio.com/docs/languages/python), a [Quarto extension](https://quarto.org/docs/tools/vscode.html), and an [AWS extension](https://aws.amazon.com/visualstudiocode/)).  
 
 ## Installing Packages 
 
-Anaconda comes with a package management system called `conda`. To install a package, open your command prompt and run 
-
-`conda install package-name`
+We recommend using the package management tool `pip` that comes with Base Python. To install a package, you can run `pip install package-name` from the command line.
 
 Note that this must be done from the command line and **not** the Python session (in contrast to package management in R).
 
-Base Python has its own package management tool called `pip`. If a package is not available with `conda install`, you can try running 
+If you are working on a computer that already has Anaconda installed (and predates the TOS changes), you can still use `conda-forge` (the open-source alternative to `conda`) to install packages. To do this, you can run:
+- `conda config --add channels conda-forge`
+- `conda install package-name`
 
-`pip install package-name`
+## Running Code Interactively
+To run Python code in Quarto notebooks or IPython notebooks (.ipynb files), we recommend installing the Jupyter package. You can do this by running `pip install jupyter` from the command line.)
 
-You should always try running `conda install` first, as this will do a better job of resolving package dependency issues.
+We also recommend installing the following from the `Extensions` section of VS Code:
+- Python
+- Jupyter
+- Quarto
 
```

Comment:
To-do: I will go through this process and make sure this isn't missing anything needed to get Python code running in VSCode in .py, .qmd or .ipynb files.